Index: snippets/text-mode/perl-mode/headx
===================================================================
--- snippets/text-mode/perl-mode/headx	(revision 1377)
+++ snippets/text-mode/perl-mode/headx	(working copy)
@@ -1,7 +1,7 @@
 #!/usr/bin/env perl
 
-# Time-stamp: <2010-05-26 15:34:27 Wednesday by ahei>
+# Time-stamp: <2011-01-17 08:52:56 Monday by lian>
 
 # @version 1.0
-# @author ahei
+# @author lian
 
Index: snippets/text-mode/sh-mode/headx
===================================================================
--- snippets/text-mode/sh-mode/headx	(revision 1377)
+++ snippets/text-mode/sh-mode/headx	(working copy)
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
-# Time-stamp: <2010-11-21 13:43:30 Sunday by taoshanwen>
+# Time-stamp: <2011-01-17 08:53:24 Monday by lian>
 
 # @version 1.0
-# @author ahei
-
+# @author lian
Index: snippets/text-mode/python-mode/headx
===================================================================
--- snippets/text-mode/python-mode/headx	(revision 1377)
+++ snippets/text-mode/python-mode/headx	(working copy)
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 
-# Time-stamp: <2010-07-17 22:26:31 Saturday by ahei>
+# Time-stamp: <2011-01-17 08:53:49 Monday by lian>
 
 # @version 1.0
-# @author ahei
+# @author lian
 
Index: .emacs
===================================================================
--- .emacs	(revision 1377)
+++ .emacs	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2011-01-21 10:55:45 Friday by taoshanwen>
+;; Time-stamp: <2011-01-26 19:43:07 Wednesday by lian>
 
 (defconst my-emacs-path           "~/emacs/" "我的emacs相关配置文件的路径")
 (defconst my-emacs-my-lisps-path  (concat my-emacs-path "my-lisps/") "我自己写的emacs lisp包的路径")
@@ -13,7 +13,7 @@
 (my-add-subdirs-to-load-path my-emacs-my-lisps-path)
 
 ;; 一个Emacs的16进制文件查看器，可以瞬间打开巨大的文件，比官方的hexl-mode好用
-(require 'hexview-mode)
+;; (require 'hexview-mode)
 
 ;; 一些基本的小函数
 (require 'ahei-misc)
@@ -62,7 +62,7 @@
 
 ;; 打开压缩文件时自动解压缩
 ;; 必须放在session前面
-(auto-compression-mode 1)
+;; (auto-compression-mode 1)
 
 ;; 所有关于buffer方面的配置
 (require 'all-buffer-settings)
@@ -137,7 +137,7 @@
 
 ;; Emacs可以做为一个server, 然后用emacsclient连接这个server,
 ;; 无需再打开两个Emacs
-(require 'emacs-server-settings)
+;; (require 'emacs-server-settings)
 
 (require 'apropos-settings)
 (require 'completion-list-mode-settings)
@@ -275,6 +275,8 @@
    ("C-x M-r" query-replace-regexp-sb)
    ("M-z"     zap-to-char-sb)
    ("C-j"     goto-line)
+   ("C-x F"   find-dired)
+   ("C-u"     cua-scroll-down)
    ("C-x C-s" save-buffer-sb)))
 
 (am-add-hooks
@@ -290,14 +292,14 @@
 
 ;; Emacs中的文本浏览器w3m
 ;; http://emacser.com/w3m.htm
-(require 'w3m-settings)
+;; (require 'w3m-settings)
 
 ;; 以另一用户编辑文件, 或者编辑远程主机文件
 (require 'tramp-settings)
 
 ;; erc: Emacs中的IRC
 ;; ERC使用简介 emacser.com/erc.htm
-(require 'erc-settings)
+;; (require 'erc-settings)
 
 ;; spell check
 (setq-default ispell-program-name "aspell")
@@ -319,9 +321,9 @@
   (add-hook 'kill-emacs-hook 'install-.emacs))
 
 ;; 把pdf,ps,dvi文件转换为png格式, 在Emacs里面浏览
-(if is-after-emacs-23
-    (require 'doc-view)
-  (setq doc-view-conversion-refresh-interval 3))
+;; (if is-after-emacs-23
+;;   (require 'doc-view)
+;;   (setq doc-view-conversion-refresh-interval 3))
 
 ;; 在Emacs里面使用shell
 (require 'term-settings)
@@ -332,7 +334,7 @@
 (require 'ioccur)
 
 ;; 查询天气预报
-(require 'weather-settings)
+;; (require 'weather-settings)
 
 ;; 可以把光标由方块变成一个小长条
 (require 'bar-cursor)
@@ -404,11 +406,11 @@
  `(("C-c M-C" copy-file-name)))
 
 ;; notification tool
-(require 'todochiku-settings)
+;; (require 'todochiku-settings)
 
 ;; twitter client
-(require 'eagle-settings)
-(require 'twit-settings)
+;; (require 'eagle-settings)
+;; (require 'twit-settings)
 
 ;; 模拟vi的点(.)命令
 (require 'dot-mode)
@@ -434,11 +436,11 @@
 (require 'org-settings)
 
 ;; 强大的发布工具
-(require 'muse-settings)
+;; (require 'muse-settings)
 
 ;; 用weblogger写WordPress博客
 ;; http://emacser.com/weblogger.htm
-(require 'weblogger-settings)
+;; (require 'weblogger-settings)
 
 ;; 关闭buffer的时候, 如果该buffer有对应的进程存在, 不提示, 烦
 (delq 'process-kill-buffer-query-function kill-buffer-query-functions)
@@ -477,7 +479,7 @@
  `(("C-x M-O" open-current-file-with-app)))
 
 ;; 一些有趣的东西
-(require 'funny)
+;; (require 'funny)
 
 ;; 模拟vi的f命令
 (require 'go-to-char)
Index: my-lisps/cedet/semantic/cedet-semantic-settings.el
===================================================================
--- my-lisps/cedet/semantic/cedet-semantic-settings.el	(revision 1377)
+++ my-lisps/cedet/semantic/cedet-semantic-settings.el	(working copy)
@@ -2,7 +2,7 @@
 
 ;; Author: ahei <ahei0802@gmail.com>
 ;; URL: http://code.google.com/p/dea/source/browse/trunk/my-lisps/cedet-semantic-settings.el
-;; Time-stamp: <2010-11-30 11:42:06 Tuesday by taoshanwen>
+;; Time-stamp: <2011-01-28 00:52:17 Friday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -21,7 +21,25 @@
 
 (semantic-load-enable-excessive-code-helpers)
 (global-semantic-idle-completions-mode -1)
-(semantic-load-enable-semantic-debugging-helpers)
+;; (semantic-load-enable-semantic-debugging-helpers)
+(setq eassist-header-switches
+      '(("h" . ("cpp" "cxx" "c++" "CC" "cc" "C" "c" "mm" "m" "hxx"))
+        ("hh" . ("cc" "CC" "cpp" "cxx" "c++" "C"))
+        ("hpp" . ("cpp" "cxx" "c++" "cc" "CC" "C"))
+        ("hxx" . ("cxx" "cpp" "c++" "cc" "CC" "C" "h"))
+        ("h++" . ("c++" "cpp" "cxx" "cc" "CC" "C"))
+        ("H" . ("C" "CC" "cc" "cpp" "cxx" "c++" "mm" "m"))
+        ("HH" . ("CC" "cc" "C" "cpp" "cxx" "c++"))
+        ("cpp" . ("hpp" "hxx" "h++" "HH" "hh" "H" "h"))
+        ("cxx" . ("hxx" "hpp" "h++" "HH" "hh" "H" "h"))
+        ("c++" . ("h++" "hpp" "hxx" "HH" "hh" "H" "h"))
+        ("CC" . ("HH" "hh" "hpp" "hxx" "h++" "H" "h"))
+        ("cc" . ("hh" "HH" "hpp" "hxx" "h++" "H" "h"))
+        ("C" . ("hpp" "hxx" "h++" "HH" "hh" "H" "h"))
+        ("c" . ("h"))
+        ("m" . ("h"))
+        ("mm" . ("h"))))
+
 (when window-system
   (global-semantic-tag-folding-mode 1))
 
@@ -38,7 +56,8 @@
      ("C-c n"   senator-next-tag)
      ("C-c p"   senator-previous-tag)
      ("C-c M-m" semantic-goto-local-main)
-     ("C-c M-M" semantic-goto-main)))
+     ("C-c M-M" semantic-goto-main)
+     ("C-c C-i" eassist-switch-h-cpp)))
 
   (defun semantic-goto-main ()
     "Jump to main function."
@@ -98,7 +117,51 @@
    'emaci-mode-map
    `(("." emaci-.)
      ("," emaci-\,)))
+
+  ;; persistent-path
+  (defconst user-persistent-path
+    (list "/glouton/struct/lian/develop/Xfem/Xfem/Xfem"
+          "/glouton/struct/lian/develop/Xext/Xext/Xext"
+          "/glouton/struct/lian/develop/Xext/Octree/Octree"
+          "/glouton/struct/lian/develop/Xext/OctreeInterface/OctreeInterface"
+          "/glouton/struct/lian/develop/SolverInterfaces/SolverBase/SolverBase"
+          "/glouton/struct/lian/develop/SolverInterfaces/SuperLu/SuperLu"
+          "/glouton/struct/lian/develop/SolverInterfaces/Lapack/Lapack"
+          "/glouton/struct/lian/develop/SolverInterfaces/Taucs/Taucs"
+          "/glouton/struct/lian/develop/SolverInterfaces/PetscSeq/PetscSeq"
+          "/glouton/struct/lian/develop/SolverInterfaces/NonLinearSolver/NonLinearSolver"
+          "/glouton/struct/lian/develop/Trellis/AOMD/AOMD/src/"
+          "/glouton/struct/lian/develop/Trellis/AOMD/Util/Util/src"
+          "/glouton/struct/lian/develop/MeshMachine/LinearAlgebra/LinearAlgebra"
+          "/glouton/struct/lian/develop/Applis/Homog/Homog"
+          "/glouton/struct/lian/develop/Applis/ITK_Utility/ITK_Utility"))
+  (setq semanticdb-persistent-path user-persistent-path)
   
+  ;; my include path
+  (defconst user-include-dirs
+    (list "/glouton/struct/lian/develop/Xfem/Xfem/include/"
+          "/glouton/struct/lian/develop/Xext/Xext/include"
+          "/glouton/struct/lian/develop/Xext/Octree/include"
+          "/glouton/struct/lian/develop/Xext/OctreeInterface/include"
+          "/glouton/struct/lian/develop/SolverInterfaces/SolverBase/include"
+          "/glouton/struct/lian/develop/SolverInterfaces/SuperLu/include"
+          "/glouton/struct/lian/develop/SolverInterfaces/Lapack/include"
+          "/glouton/struct/lian/develop/SolverInterfaces/Taucs/include"
+          "/glouton/struct/lian/develop/SolverInterfaces/PetscSeq/include"
+          "/glouton/struct/lian/develop/SolverInterfaces/NonLinearSolver/include"
+          "/glouton/struct/lian/develop/Trellis/AOMD/include"
+          "/glouton/struct/lian/develop/Trellis/Util/include"
+          "/glouton/struct/lian/develop/Trellis/model/include"
+          "/glouton/struct/lian/develop/MeshMachine/LinearAlgebra/include"
+          "/glouton/struct/lian/develop/Applis/Homog/include"
+          "/glouton/struct/lian/develop/Applis/Physics/include"
+          "/glouton/struct/lian/develop/Applis/ITK_Utility/include"))
+  (let ((include-dirs user-include-dirs))
+    (mapc (lambda (dir)
+            (semantic-add-system-include dir 'c++-mode)
+            (semantic-add-system-include dir 'c-mode))
+          include-dirs))
+  
   ;; system include path
   (if (or mswin cygwin)
       (dolist (mode '(c-mode c++-mode))
Index: my-lisps/color-theme-ahei.el
===================================================================
--- my-lisps/color-theme-ahei.el	(revision 1377)
+++ my-lisps/color-theme-ahei.el	(working copy)
@@ -4,7 +4,7 @@
 
 ;; Author: ahei <ahei0802@126.com>
 ;; Keywords: color theme ahei
-;; Time-stamp: <2010-04-05 15:56:28 Monday by ahei>
+;; Time-stamp: <2011-01-27 10:51:22 Thursday by lian>
 
 ;; This file is free software; you can redistribute it and/or modify
 ;; it under the terms of the GNU General Public License as published by
Index: my-lisps/font-settings.el
===================================================================
--- my-lisps/font-settings.el	(revision 1377)
+++ my-lisps/font-settings.el	(working copy)
@@ -9,6 +9,6 @@
       (huangq-fontset-consolas 17)
     ;; (huangq-fontset-dejavu 17)))
     ;; (huangq-fontset-fixedsys 17)
-    (set-default-font "Monospace 11")))
+    (set-default-font "Monospace 9")))
 
 (provide 'font-settings)
Index: my-lisps/vc-settings.el
===================================================================
--- my-lisps/vc-settings.el	(revision 1377)
+++ my-lisps/vc-settings.el	(working copy)
@@ -2,7 +2,7 @@
 
 ;; Author: ahei <ahei0802@gmail.com>
 ;; URL: http://code.google.com/p/dea/source/browse/trunk/my-lisps/vc-settings.el
-;; Time-stamp: <2011-01-21 10:54:35 Friday by taoshanwen>
+;; Time-stamp: <2011-01-16 22:54:37 Sunday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -57,7 +57,8 @@
 
 (defun vc-settings ()
   "Settings for `vc'."
-  (require 'vc+))
+  (require 'vc+)
+  (setq vc-follow-symlinks t))
 
 (eval-after-load "vc-svn"
   `(vc-settings))
Index: my-lisps/misc-settings.el
===================================================================
--- my-lisps/misc-settings.el	(revision 1377)
+++ my-lisps/misc-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-11-30 09:55:29 Tuesday by taoshanwen>
+;; Time-stamp: <2010-12-06 15:15:53 Monday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -18,8 +18,8 @@
 ;; Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
 ;; 个人信息
-(setq user-mail-address "ahei0802@gmail.com")
-(setq user-full-name    "ahei")
+(setq user-mail-address "weidong.lian@gmail.com")
+(setq user-full-name    "lianweidong")
 
 (setq-default default-directory "~")
 
Index: my-lisps/isearch-face-settings.el
===================================================================
--- my-lisps/isearch-face-settings.el	(revision 1377)
+++ my-lisps/isearch-face-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-04-05 17:06:00 Monday by ahei>
+;; Time-stamp: <2011-01-26 18:49:13 Wednesday by lian>
 
 ;; This file is free software; you can redistribute it and/or modify
 ;; it under the terms of the GNU General Public License as published by
@@ -23,7 +23,7 @@
   (set-face-background 'isearch "blue")
   (when (not is-before-emacs-21)
     (set-face-foreground 'lazy-highlight "black")
-    (set-face-background 'lazy-highlight "white"))
+    (set-face-background 'lazy-highlight "yellow"))
   (custom-set-faces '(isearch-fail ((((class color)) (:background "red"))))))
 
 (eval-after-load "isearch"
Index: my-lisps/dev-settings.el
===================================================================
--- my-lisps/dev-settings.el	(revision 1377)
+++ my-lisps/dev-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-11-29 14:42:43 Monday by taoshanwen>
+;; Time-stamp: <2010-12-16 12:21:47 Thursday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -52,7 +52,7 @@
  `(("{" skeleton-c-mode-left-brace)))
 
 ;; 动态检查语法错误
-(require 'flymake-settings)
+;; (require 'flymake-settings)
 
 (defun skeleton-c-mode-left-brace (arg)
   (interactive "P")
@@ -174,6 +174,7 @@
      (message (concat "Wrote " (buffer-file-name))))))
 
 ;; 显示变量, 函数的声明
+
 (require 'eldoc-settings)
 
 ;; 自动给你加上括号
