Index: snippets/text-mode/perl-mode/headx
===================================================================
--- snippets/text-mode/perl-mode/headx	(revision 1377)
+++ snippets/text-mode/perl-mode/headx	(working copy)
@@ -1,7 +1,7 @@
 #!/usr/bin/env perl
 
-# Time-stamp: <2010-05-26 15:34:27 Wednesday by ahei>
+# Time-stamp: <2011-01-17 08:52:56 Monday by lian>
 
 # @version 1.0
-# @author ahei
+# @author lian
 
Index: snippets/text-mode/sh-mode/headx
===================================================================
--- snippets/text-mode/sh-mode/headx	(revision 1377)
+++ snippets/text-mode/sh-mode/headx	(working copy)
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
-# Time-stamp: <2010-11-21 13:43:30 Sunday by taoshanwen>
+# Time-stamp: <2011-01-17 08:53:24 Monday by lian>
 
 # @version 1.0
-# @author ahei
-
+# @author lian
Index: snippets/text-mode/python-mode/headx
===================================================================
--- snippets/text-mode/python-mode/headx	(revision 1377)
+++ snippets/text-mode/python-mode/headx	(working copy)
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 
-# Time-stamp: <2010-07-17 22:26:31 Saturday by ahei>
+# Time-stamp: <2011-01-17 08:53:49 Monday by lian>
 
 # @version 1.0
-# @author ahei
+# @author lian
 
Index: .emacs
===================================================================
--- .emacs	(revision 1377)
+++ .emacs	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2011-01-21 10:55:45 Friday by taoshanwen>
+;; Time-stamp: <2011-02-08 01:59:42 Tuesday by lian>
 
 (defconst my-emacs-path           "~/emacs/" "我的emacs相关配置文件的路径")
 (defconst my-emacs-my-lisps-path  (concat my-emacs-path "my-lisps/") "我自己写的emacs lisp包的路径")
@@ -12,8 +12,11 @@
 (my-add-subdirs-to-load-path my-emacs-lisps-path)
 (my-add-subdirs-to-load-path my-emacs-my-lisps-path)
 
+;; lian-info
+(require 'lian-info)
+
 ;; 一个Emacs的16进制文件查看器，可以瞬间打开巨大的文件，比官方的hexl-mode好用
-(require 'hexview-mode)
+;; (require 'hexview-mode)
 
 ;; 一些基本的小函数
 (require 'ahei-misc)
@@ -62,7 +65,7 @@
 
 ;; 打开压缩文件时自动解压缩
 ;; 必须放在session前面
-(auto-compression-mode 1)
+;; (auto-compression-mode 1)
 
 ;; 所有关于buffer方面的配置
 (require 'all-buffer-settings)
@@ -137,7 +140,7 @@
 
 ;; Emacs可以做为一个server, 然后用emacsclient连接这个server,
 ;; 无需再打开两个Emacs
-(require 'emacs-server-settings)
+;; (require 'emacs-server-settings)
 
 (require 'apropos-settings)
 (require 'completion-list-mode-settings)
@@ -275,6 +278,9 @@
    ("C-x M-r" query-replace-regexp-sb)
    ("M-z"     zap-to-char-sb)
    ("C-j"     goto-line)
+   ("C-x F"   find-dired)
+   ("C-u"     cua-scroll-down)
+   ("<f9>"    kill-this-buffer)
    ("C-x C-s" save-buffer-sb)))
 
 (am-add-hooks
@@ -290,14 +296,14 @@
 
 ;; Emacs中的文本浏览器w3m
 ;; http://emacser.com/w3m.htm
-(require 'w3m-settings)
+;; (require 'w3m-settings)
 
 ;; 以另一用户编辑文件, 或者编辑远程主机文件
 (require 'tramp-settings)
 
 ;; erc: Emacs中的IRC
 ;; ERC使用简介 emacser.com/erc.htm
-(require 'erc-settings)
+;; (require 'erc-settings)
 
 ;; spell check
 (setq-default ispell-program-name "aspell")
@@ -319,9 +325,9 @@
   (add-hook 'kill-emacs-hook 'install-.emacs))
 
 ;; 把pdf,ps,dvi文件转换为png格式, 在Emacs里面浏览
-(if is-after-emacs-23
-    (require 'doc-view)
-  (setq doc-view-conversion-refresh-interval 3))
+;; (if is-after-emacs-23
+;;   (require 'doc-view)
+;;   (setq doc-view-conversion-refresh-interval 3))
 
 ;; 在Emacs里面使用shell
 (require 'term-settings)
@@ -332,7 +338,7 @@
 (require 'ioccur)
 
 ;; 查询天气预报
-(require 'weather-settings)
+;; (require 'weather-settings)
 
 ;; 可以把光标由方块变成一个小长条
 (require 'bar-cursor)
@@ -407,8 +413,8 @@
 (require 'todochiku-settings)
 
 ;; twitter client
-(require 'eagle-settings)
-(require 'twit-settings)
+;; (require 'eagle-settings)
+;; (require 'twit-settings)
 
 ;; 模拟vi的点(.)命令
 (require 'dot-mode)
@@ -434,11 +440,11 @@
 (require 'org-settings)
 
 ;; 强大的发布工具
-(require 'muse-settings)
+;; (require 'muse-settings)
 
 ;; 用weblogger写WordPress博客
 ;; http://emacser.com/weblogger.htm
-(require 'weblogger-settings)
+;; (require 'weblogger-settings)
 
 ;; 关闭buffer的时候, 如果该buffer有对应的进程存在, 不提示, 烦
 (delq 'process-kill-buffer-query-function kill-buffer-query-functions)
@@ -477,7 +483,7 @@
  `(("C-x M-O" open-current-file-with-app)))
 
 ;; 一些有趣的东西
-(require 'funny)
+;; (require 'funny)
 
 ;; 模拟vi的f命令
 (require 'go-to-char)
@@ -504,4 +510,4 @@
 (sb-update)
 
 ;; 加载用户自己的配置
-(load "~/my.emacs" t)
+(load "~/my.emacs" t) 
Index: my-lisps/cedet/semantic/cedet-semantic-settings.el
===================================================================
--- my-lisps/cedet/semantic/cedet-semantic-settings.el	(revision 1377)
+++ my-lisps/cedet/semantic/cedet-semantic-settings.el	(working copy)
@@ -2,7 +2,7 @@
 
 ;; Author: ahei <ahei0802@gmail.com>
 ;; URL: http://code.google.com/p/dea/source/browse/trunk/my-lisps/cedet-semantic-settings.el
-;; Time-stamp: <2010-11-30 11:42:06 Tuesday by taoshanwen>
+;; Time-stamp: <2011-02-08 02:27:02 Tuesday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -21,7 +21,8 @@
 
 (semantic-load-enable-excessive-code-helpers)
 (global-semantic-idle-completions-mode -1)
-(semantic-load-enable-semantic-debugging-helpers)
+;; (semantic-load-enable-semantic-debugging-helpers)
+(semantic-load-enable-all-exuberent-ctags-support) 
 (when window-system
   (global-semantic-tag-folding-mode 1))
 
@@ -98,7 +99,18 @@
    'emaci-mode-map
    `(("." emaci-.)
      ("," emaci-\,)))
+
+  ;; semantic persistent-path  
+  (setq semanticdb-persistent-path (append semanticdb-persistent-path lian-source-file-dir))
   
+  ;; my include path
+  (let ((include-dirs lian-head-file-dir))
+    (mapc (lambda (dir)
+            (semantic-add-system-include dir 'c++-mode)
+            (semantic-add-system-include dir 'c-mode))
+          include-dirs))
+  
+  
   ;; system include path
   (if (or mswin cygwin)
       (dolist (mode '(c-mode c++-mode))
Index: my-lisps/edit-settings.el
===================================================================
--- my-lisps/edit-settings.el	(revision 1377)
+++ my-lisps/edit-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-09-02 17:09:59 Thursday by taoshanwen>
+;; Time-stamp: <2011-01-28 09:57:56 Friday by lian>
 
 (require 'compile-settings)
 (require 'edit-misc)
@@ -55,7 +55,8 @@
                     conf-javaprop-mode-map html-mode-map tcl-mode-map
                     autoconf-mode-map perl-mode-map nxml-mode-map
                     python-mode-map))
- `(("C-c C-c" comment)))
+ `(("C-c C-c" comment)
+   ("C-c C-x" uncomment)))
 
 ;;;###autoload
 (defmacro def-redo-command (fun-name redo undo)
Index: my-lisps/sourcepair-settings.el
===================================================================
--- my-lisps/sourcepair-settings.el	(revision 1377)
+++ my-lisps/sourcepair-settings.el	(working copy)
@@ -2,7 +2,7 @@
 
 ;; Author: ahei <ahei0802@gmail.com>
 ;; URL: http://code.google.com/p/dea/source/browse/trunk/my-lisps/sourcepair-settings.el
-;; Time-stamp: <2010-04-11 19:13:11 Sunday by ahei>
+;; Time-stamp: <2011-01-29 02:06:06 Saturday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -21,7 +21,8 @@
 
 (eal-define-keys
  `(c-mode-base-map)
- `(("C-c s" sourcepair-load)))
+ `(("C-c s" sourcepair-load)
+   ("C-c C-i" sourcepair-load)))
 
 (autoload 'sourcepair-load "sourcepair"
   "Load the corresponding C/C++ header or source file for the current buffer.
@@ -47,7 +48,9 @@
 (defun sourcepair-settings ()
   "Settings for `sourcepair'."
   (setq sourcepair-source-path '( "." "../src"))
+  (setq sourcepair-source-path (append sourcepair-source-path lian-source-file-dir))
   (setq sourcepair-header-path user-head-file-dir)
+  (setq sourcepair-source-extensions (append sourcepair-source-extensions '(".txx")))
   (setq sourcepair-recurse-ignore '("CVS" "bin" "lib" "Obj" "Debug" "Release" ".svn")))
 
 (eval-after-load "sourcepair"
Index: my-lisps/c-settings.el
===================================================================
--- my-lisps/c-settings.el	(revision 1377)
+++ my-lisps/c-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-08-29 18:34:06 Sunday by taoshanwen>
+;; Time-stamp: <2011-01-29 02:17:18 Saturday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -43,6 +43,7 @@
   (add-hook 'c-mode-common-hook 'c-mode-common-hook-settings)
 
   (add-to-list 'auto-mode-alist '("\\.h$" . c++-mode))
+  (add-to-list 'auto-mode-alist '("\\.txx$" . c++-mode))
 
   ;; kde-emacs 一个方便开发c的工具
   (require 'kde-emacs-settings)
Index: my-lisps/color-theme-ahei.el
===================================================================
--- my-lisps/color-theme-ahei.el	(revision 1377)
+++ my-lisps/color-theme-ahei.el	(working copy)
@@ -4,7 +4,7 @@
 
 ;; Author: ahei <ahei0802@126.com>
 ;; Keywords: color theme ahei
-;; Time-stamp: <2010-04-05 15:56:28 Monday by ahei>
+;; Time-stamp: <2011-01-27 10:51:22 Thursday by lian>
 
 ;; This file is free software; you can redistribute it and/or modify
 ;; it under the terms of the GNU General Public License as published by
Index: my-lisps/font-settings.el
===================================================================
--- my-lisps/font-settings.el	(revision 1377)
+++ my-lisps/font-settings.el	(working copy)
@@ -9,6 +9,6 @@
       (huangq-fontset-consolas 17)
     ;; (huangq-fontset-dejavu 17)))
     ;; (huangq-fontset-fixedsys 17)
-    (set-default-font "Monospace 11")))
+    (set-default-font "Monospace 9")))
 
 (provide 'font-settings)
Index: my-lisps/vc-settings.el
===================================================================
--- my-lisps/vc-settings.el	(revision 1377)
+++ my-lisps/vc-settings.el	(working copy)
@@ -2,7 +2,7 @@
 
 ;; Author: ahei <ahei0802@gmail.com>
 ;; URL: http://code.google.com/p/dea/source/browse/trunk/my-lisps/vc-settings.el
-;; Time-stamp: <2011-01-21 10:54:35 Friday by taoshanwen>
+;; Time-stamp: <2011-01-28 01:23:46 Friday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -23,8 +23,7 @@
  `(emacs-lisp-mode-map c-mode-base-map sgml-mode-map
                        sh-mode-map text-mode-map conf-javaprop-mode-map
                        c++-mode-map image-mode-map makefile-gmake-mode-map
-                       org-mode-map nxml-mode-map python-mode-map perl-mode-map
-                       markdown-mode-map)
+                       org-mode-map nxml-mode-map python-mode-map perl-mode-map)
  `(("C-c L"   vc-print-log)
    ("C-c C-u" vc-update-and-revert-buffer)
    ("C-c C-b" vc-annotate)
@@ -57,6 +56,7 @@
 
 (defun vc-settings ()
   "Settings for `vc'."
+  (setq vc-follow-symlinks t)
   (require 'vc+))
 
 (eval-after-load "vc-svn"
Index: my-lisps/misc-settings.el
===================================================================
--- my-lisps/misc-settings.el	(revision 1377)
+++ my-lisps/misc-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-11-30 09:55:29 Tuesday by taoshanwen>
+;; Time-stamp: <2010-12-06 15:15:53 Monday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -18,8 +18,8 @@
 ;; Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
 ;; 个人信息
-(setq user-mail-address "ahei0802@gmail.com")
-(setq user-full-name    "ahei")
+(setq user-mail-address "weidong.lian@gmail.com")
+(setq user-full-name    "lianweidong")
 
 (setq-default default-directory "~")
 
Index: my-lisps/isearch-face-settings.el
===================================================================
--- my-lisps/isearch-face-settings.el	(revision 1377)
+++ my-lisps/isearch-face-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-04-05 17:06:00 Monday by ahei>
+;; Time-stamp: <2011-01-28 09:18:32 Friday by lian>
 
 ;; This file is free software; you can redistribute it and/or modify
 ;; it under the terms of the GNU General Public License as published by
@@ -23,7 +23,7 @@
   (set-face-background 'isearch "blue")
   (when (not is-before-emacs-21)
     (set-face-foreground 'lazy-highlight "black")
-    (set-face-background 'lazy-highlight "white"))
+    (set-face-background 'lazy-highlight "yellow"))
   (custom-set-faces '(isearch-fail ((((class color)) (:background "red"))))))
 
 (eval-after-load "isearch"
Index: my-lisps/compile-settings.el
===================================================================
--- my-lisps/compile-settings.el	(revision 1377)
+++ my-lisps/compile-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-11-21 15:47:18 Sunday by taoshanwen>
+;; Time-stamp: <2011-01-28 23:07:30 Friday by lian>
 
 (require 'util)
 
@@ -41,7 +41,7 @@
 (defun compile-settings ()
   "Settings for `compile'."
   ;; 设置编译命令
-  (setq compile-command "make -k")
+  (setq compile-command "make -k -j 8 NODEP=1 VERS=opt")
 
   (eal-define-keys
    makefile-mode-map-list
Index: my-lisps/dev-settings.el
===================================================================
--- my-lisps/dev-settings.el	(revision 1377)
+++ my-lisps/dev-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-11-29 14:42:43 Monday by taoshanwen>
+;; Time-stamp: <2010-12-16 12:21:47 Thursday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -52,7 +52,7 @@
  `(("{" skeleton-c-mode-left-brace)))
 
 ;; 动态检查语法错误
-(require 'flymake-settings)
+;; (require 'flymake-settings)
 
 (defun skeleton-c-mode-left-brace (arg)
   (interactive "P")
@@ -174,6 +174,7 @@
      (message (concat "Wrote " (buffer-file-name))))))
 
 ;; 显示变量, 函数的声明
+
 (require 'eldoc-settings)
 
 ;; 自动给你加上括号
