Index: snippets/text-mode/perl-mode/headx
===================================================================
--- snippets/text-mode/perl-mode/headx	(revision 1384)
+++ snippets/text-mode/perl-mode/headx	(working copy)
@@ -1,7 +1,7 @@
 #!/usr/bin/env perl
 
-# Time-stamp: <2010-05-26 15:34:27 Wednesday by ahei>
+# Time-stamp: <2011-01-17 08:52:56 Monday by lian>
 
 # @version 1.0
-# @author ahei
+# @author lian
 
Index: snippets/text-mode/sh-mode/headx
===================================================================
--- snippets/text-mode/sh-mode/headx	(revision 1384)
+++ snippets/text-mode/sh-mode/headx	(working copy)
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
-# Time-stamp: <2010-11-21 13:43:30 Sunday by taoshanwen>
+# Time-stamp: <2011-01-17 08:53:24 Monday by lian>
 
 # @version 1.0
-# @author ahei
-
+# @author lian
Index: snippets/text-mode/python-mode/headx
===================================================================
--- snippets/text-mode/python-mode/headx	(revision 1384)
+++ snippets/text-mode/python-mode/headx	(working copy)
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 
-# Time-stamp: <2010-07-17 22:26:31 Saturday by ahei>
+# Time-stamp: <2011-01-17 08:53:49 Monday by lian>
 
 # @version 1.0
-# @author ahei
+# @author lian
 
Index: .emacs
===================================================================
--- .emacs	(revision 1384)
+++ .emacs	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2011-03-17 19:58:55 Thursday by taoshanwen>
+;; Time-stamp: <2011-04-07 14:10:06 Thursday by lian>
 
 (defconst my-emacs-path           "~/emacs/" "我的emacs相关配置文件的路径")
 (defconst my-emacs-my-lisps-path  (concat my-emacs-path "my-lisps/") "我自己写的emacs lisp包的路径")
@@ -12,8 +12,11 @@
 (my-add-subdirs-to-load-path my-emacs-lisps-path)
 (my-add-subdirs-to-load-path my-emacs-my-lisps-path)
 
+;; lian-info
+(require 'lian-info)
+
 ;; 一个Emacs的16进制文件查看器，可以瞬间打开巨大的文件，比官方的hexl-mode好用
-(require 'hexview-mode)
+;; (require 'hexview-mode)
 
 ;; 一些基本的小函数
 (require 'ahei-misc)
@@ -62,7 +65,7 @@
 
 ;; 打开压缩文件时自动解压缩
 ;; 必须放在session前面
-(auto-compression-mode 1)
+;; (auto-compression-mode 1)
 
 ;; 所有关于buffer方面的配置
 (require 'all-buffer-settings)
@@ -137,7 +140,7 @@
 
 ;; Emacs可以做为一个server, 然后用emacsclient连接这个server,
 ;; 无需再打开两个Emacs
-(require 'emacs-server-settings)
+;; (require 'emacs-server-settings)
 
 (require 'apropos-settings)
 (require 'completion-list-mode-settings)
@@ -274,7 +277,10 @@
    ("ESC M-r" query-replace-regexp-sb)
    ("C-x M-r" query-replace-regexp-sb)
    ("M-z"     zap-to-char-sb)
-   ("C-j"     goto-line)
+   ("C-M-j"     goto-line)
+   ("C-x F"   find-dired)
+   ("C-u"     cua-scroll-down)
+   ("<f9>"    kill-this-buffer)
    ("C-x C-s" save-buffer-sb)))
 
 (am-add-hooks
@@ -297,7 +303,7 @@
 
 ;; erc: Emacs中的IRC
 ;; ERC使用简介 emacser.com/erc.htm
-(require 'erc-settings)
+;; (require 'erc-settings)
 
 ;; spell check
 (setq-default ispell-program-name "aspell")
@@ -319,9 +325,9 @@
   (add-hook 'kill-emacs-hook 'install-.emacs))
 
 ;; 把pdf,ps,dvi文件转换为png格式, 在Emacs里面浏览
-(if is-after-emacs-23
-    (require 'doc-view)
-  (setq doc-view-conversion-refresh-interval 3))
+;; (if is-after-emacs-23
+;;   (require 'doc-view)
+;;   (setq doc-view-conversion-refresh-interval 3))
 
 ;; 在Emacs里面使用shell
 (require 'term-settings)
@@ -332,7 +338,7 @@
 (require 'ioccur)
 
 ;; 查询天气预报
-(require 'weather-settings)
+;; (require 'weather-settings)
 
 ;; 可以把光标由方块变成一个小长条
 (require 'bar-cursor)
@@ -410,8 +416,8 @@
 (require 'todochiku-settings)
 
 ;; twitter client
-(require 'eagle-settings)
-(require 'twit-settings)
+;; (require 'eagle-settings)
+;; (require 'twit-settings)
 
 ;; 模拟vi的点(.)命令
 (require 'dot-mode)
@@ -437,11 +443,11 @@
 (require 'org-settings)
 
 ;; 强大的发布工具
-(require 'muse-settings)
+;; (require 'muse-settings)
 
 ;; 用weblogger写WordPress博客
 ;; http://emacser.com/weblogger.htm
-(require 'weblogger-settings)
+;; (require 'weblogger-settings)
 
 ;; 关闭buffer的时候, 如果该buffer有对应的进程存在, 不提示, 烦
 (delq 'process-kill-buffer-query-function kill-buffer-query-functions)
@@ -480,7 +486,7 @@
  `(("C-x M-O" open-current-file-with-app)))
 
 ;; 一些有趣的东西
-(require 'funny)
+;; (require 'funny)
 
 ;; 模拟vi的f命令
 (require 'go-to-char)
@@ -507,4 +513,4 @@
 (sb-update)
 
 ;; 加载用户自己的配置
-(load "~/my.emacs" t)
+(load "~/my.emacs" t) 
Index: my-lisps/cedet/semantic/cedet-semantic-settings.el
===================================================================
--- my-lisps/cedet/semantic/cedet-semantic-settings.el	(revision 1384)
+++ my-lisps/cedet/semantic/cedet-semantic-settings.el	(working copy)
@@ -2,7 +2,7 @@
 
 ;; Author: ahei <ahei0802@gmail.com>
 ;; URL: http://code.google.com/p/dea/source/browse/trunk/my-lisps/cedet-semantic-settings.el
-;; Time-stamp: <2010-11-30 11:42:06 Tuesday by taoshanwen>
+;; Time-stamp: <2011-02-18 18:54:57 Friday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -21,7 +21,8 @@
 
 (semantic-load-enable-excessive-code-helpers)
 (global-semantic-idle-completions-mode -1)
-(semantic-load-enable-semantic-debugging-helpers)
+;; (semantic-load-enable-semantic-debugging-helpers)
+;; (semantic-load-enable-all-exuberent-ctags-support) 
 (when window-system
   (global-semantic-tag-folding-mode 1))
 
@@ -35,6 +36,7 @@
    `(c-mode-base-map makefile-gmake-mode-map python-mode-map perl-mode-map sh-mode-map)
    `(("C-c C-j" semantic-ia-fast-jump)
      ("C-c j"   semantic-complete-jump-local)
+     ("C-c u" semantic-debug-idle-work-function)
      ("C-c n"   senator-next-tag)
      ("C-c p"   senator-previous-tag)
      ("C-c M-m" semantic-goto-local-main)
@@ -98,7 +100,18 @@
    'emaci-mode-map
    `(("." emaci-.)
      ("," emaci-\,)))
+
+  ;; semantic persistent-path  
+  (setq semanticdb-persistent-path (append semanticdb-persistent-path lian-source-file-dir))
   
+  ;; my include path
+  (let ((include-dirs lian-head-file-dir))
+    (mapc (lambda (dir)
+            (semantic-add-system-include dir 'c++-mode)
+            (semantic-add-system-include dir 'c-mode))
+          include-dirs))
+  
+  
   ;; system include path
   (if (or mswin cygwin)
       (dolist (mode '(c-mode c++-mode))
Index: my-lisps/dired-settings.el
===================================================================
--- my-lisps/dired-settings.el	(revision 1384)
+++ my-lisps/dired-settings.el	(working copy)
@@ -2,7 +2,7 @@
 
 ;; Author: ahei <ahei0802@gmail.com>
 ;; URL: http://code.google.com/p/dea/source/browse/trunk/my-lisps/dired-settings.el
-;; Time-stamp: <2010-11-21 21:15:20 Sunday by taoshanwen>
+;; Time-stamp: <2011-04-13 21:02:28 Wednesday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -30,7 +30,7 @@
   "Settings for `dired'."
 
   (setq truncate-lines t)
-
+  (setq dired-listing-switches "-alh")
   (require 'wuxch-dired "my-wuxch-dired")
   (require 'wuxch-dired-copy-paste "my-wuxch-dired-copy-paste")
 
@@ -204,7 +204,7 @@
        ("/"           dired-slash-map)
        ("/m"          ywb-dired-filter-regexp)
        ("/."          ywb-dired-filter-extension)
-       ("C-q"         ywb-dired-quickview)
+       ("C-q"         (lambda () (interactive)(if dired-lis-mode (dired-lis-off) (ywb-dired-quickview))))
        ("r"           wdired-change-to-wdired-mode)
        ("C-s"         dired-lis-isearch-forward-always?)
        ("C-r"         dired-lis-isearch-backward-always?)
@@ -235,6 +235,9 @@
        ("w"           dired-scroll-down)
        ("M->"         wuxch-dired-goto-last-line)
        ("M-<"         wuxch-dired-goto-first-line)
+       ("e"           dired-smart-shell-command)
+       ("x"           dired-do-shell-command)
+       ("X"           dired-do-flagged-delete)
 
        ("M-w"         wuxch-dired-copy)
        ("C-y"         wuxch-dired-paste)
@@ -250,6 +253,8 @@
        ("C-c m"       make-check-sb)
        ("C-c M"       make-clean-sb)
        ("C-c c"       compile-buffer-sb)
+       ("C-c C-d"     dired-smart-shell-command)
+       ("C-c C-r"     (lambda () (interactive)(find-dired default-directory "-name '#*#' -or -name '*~'")))
        ("C-c r"       run-program-sb)
        ("C-c C"       smart-compile-sb)
        ("C-c g"       gdb)
Index: my-lisps/edit-settings.el
===================================================================
--- my-lisps/edit-settings.el	(revision 1384)
+++ my-lisps/edit-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-09-02 17:09:59 Thursday by taoshanwen>
+;; Time-stamp: <2011-05-27 15:18:36 Friday by lian>
 
 (require 'compile-settings)
 (require 'edit-misc)
@@ -55,7 +55,8 @@
                     conf-javaprop-mode-map html-mode-map tcl-mode-map
                     autoconf-mode-map perl-mode-map nxml-mode-map
                     python-mode-map))
- `(("C-c C-c" comment)))
+ `(("C-c C-c" comment)
+   ("C-c C-x" uncomment)))
 
 ;;;###autoload
 (defmacro def-redo-command (fun-name redo undo)
@@ -88,7 +89,7 @@
    ("C-x M-M" mark-invisible-region)
    ("M-U" del-to-begin)
    ("C-^" case-trans)
-   ("C-6" case-trans)
+   ;; ("C-6" case-trans)
    ("C-w" backward-kill-word-or-kill-region)
    ("C-x S" mark-whole-sexp)
    ("C-x W" kill-whole-sexp)
Index: my-lisps/select-buffer.el
===================================================================
--- my-lisps/select-buffer.el	(revision 1384)
+++ my-lisps/select-buffer.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*- 
 
-;; Time-stamp: <2010-12-01 11:58:13 Wednesday by taoshanwen>
+;; Time-stamp: <2011-02-15 13:36:08 Tuesday by lian>
 
 ;;; selecte-buffer.el --- Select buffer like Alt-tab on linux system
 
@@ -48,6 +48,8 @@
  global-map
  `(("M-N"   select-buffer-forward)
    ("M-P"   select-buffer-backward)
+   ("<f10>" select-buffer-forward)
+   ("<f11>" select-buffer-backward)
    ("C-x t" sb-toggle-keep-buffer)
    ("M-'"   switch-to-other-buffer)))
 
Index: my-lisps/sourcepair-settings.el
===================================================================
--- my-lisps/sourcepair-settings.el	(revision 1384)
+++ my-lisps/sourcepair-settings.el	(working copy)
@@ -2,7 +2,7 @@
 
 ;; Author: ahei <ahei0802@gmail.com>
 ;; URL: http://code.google.com/p/dea/source/browse/trunk/my-lisps/sourcepair-settings.el
-;; Time-stamp: <2010-04-11 19:13:11 Sunday by ahei>
+;; Time-stamp: <2011-01-29 02:06:06 Saturday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -21,7 +21,8 @@
 
 (eal-define-keys
  `(c-mode-base-map)
- `(("C-c s" sourcepair-load)))
+ `(("C-c s" sourcepair-load)
+   ("C-c C-i" sourcepair-load)))
 
 (autoload 'sourcepair-load "sourcepair"
   "Load the corresponding C/C++ header or source file for the current buffer.
@@ -47,7 +48,9 @@
 (defun sourcepair-settings ()
   "Settings for `sourcepair'."
   (setq sourcepair-source-path '( "." "../src"))
+  (setq sourcepair-source-path (append sourcepair-source-path lian-source-file-dir))
   (setq sourcepair-header-path user-head-file-dir)
+  (setq sourcepair-source-extensions (append sourcepair-source-extensions '(".txx")))
   (setq sourcepair-recurse-ignore '("CVS" "bin" "lib" "Obj" "Debug" "Release" ".svn")))
 
 (eval-after-load "sourcepair"
Index: my-lisps/c-settings.el
===================================================================
--- my-lisps/c-settings.el	(revision 1384)
+++ my-lisps/c-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-08-29 18:34:06 Sunday by taoshanwen>
+;; Time-stamp: <2011-02-25 17:59:41 Friday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -28,6 +28,8 @@
  `(("C-h"     c-electric-backspace)
    ("C-c M-a" beginning-of-defun)
    ("C-c M-e" end-of-defun)
+   ("C-c C-j" find-tag)
+   ("C-c j" (lambda () (interactive)(find-tag "" t)))
    ("C-c M-F" copy-current-fun-name)))
 
 (defun cc-mode-settings ()
@@ -43,6 +45,7 @@
   (add-hook 'c-mode-common-hook 'c-mode-common-hook-settings)
 
   (add-to-list 'auto-mode-alist '("\\.h$" . c++-mode))
+  (add-to-list 'auto-mode-alist '("\\.txx$" . c++-mode))
 
   ;; kde-emacs 一个方便开发c的工具
   (require 'kde-emacs-settings)
Index: my-lisps/font-settings.el
===================================================================
--- my-lisps/font-settings.el	(revision 1384)
+++ my-lisps/font-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-10-11 20:00:55 Monday by taoshanwen>
+;; Time-stamp: <2011-05-26 10:30:54 Thursday by lian>
 
 (when (and window-system is-after-emacs-23)
   (require 'my-fontset-win)
Index: my-lisps/ido-settings.el
===================================================================
--- my-lisps/ido-settings.el	(revision 1384)
+++ my-lisps/ido-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-08-08 12:04:13 Sunday by taoshanwen>
+;; Time-stamp: <2011-02-26 17:28:16 Saturday by lian>
 
 (if is-before-emacs-21
     (require 'ido "ido-for-21"))
@@ -19,6 +19,11 @@
   (global-set-key (kbd "C-x C-f") 'ido-find-file)
   (setq ido-max-directory-size 1000000)
 
+  ;; disable auto searching for files unless called explicitly
+  (setq ido-auto-merge-delay-time 99999)
+  (define-key ido-file-dir-completion-map (kbd "C-c C-s") 
+    (lambda() (interactive) (ido-initiate-auto-merge (current-buffer))))
+  
   (defmacro def-ido-enter-command (command)
     "Make definition of command which execute some command in ido."
     `(defun ,(am-intern "ido-enter-" command) ()
Index: my-lisps/vc-settings.el
===================================================================
--- my-lisps/vc-settings.el	(revision 1384)
+++ my-lisps/vc-settings.el	(working copy)
@@ -2,7 +2,7 @@
 
 ;; Author: ahei <ahei0802@gmail.com>
 ;; URL: http://code.google.com/p/dea/source/browse/trunk/my-lisps/vc-settings.el
-;; Time-stamp: <2011-01-21 10:54:35 Friday by taoshanwen>
+;; Time-stamp: <2011-01-28 01:23:46 Friday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -23,8 +23,7 @@
  `(emacs-lisp-mode-map c-mode-base-map sgml-mode-map
                        sh-mode-map text-mode-map conf-javaprop-mode-map
                        c++-mode-map image-mode-map makefile-gmake-mode-map
-                       org-mode-map nxml-mode-map python-mode-map perl-mode-map
-                       markdown-mode-map)
+                       org-mode-map nxml-mode-map python-mode-map perl-mode-map)
  `(("C-c L"   vc-print-log)
    ("C-c C-u" vc-update-and-revert-buffer)
    ("C-c C-b" vc-annotate)
@@ -57,6 +56,7 @@
 
 (defun vc-settings ()
   "Settings for `vc'."
+  (setq vc-follow-symlinks t)
   (require 'vc+))
 
 (eval-after-load "vc-svn"
Index: my-lisps/misc-settings.el
===================================================================
--- my-lisps/misc-settings.el	(revision 1384)
+++ my-lisps/misc-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-11-30 09:55:29 Tuesday by taoshanwen>
+;; Time-stamp: <2010-12-06 15:15:53 Monday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -18,8 +18,8 @@
 ;; Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
 ;; 个人信息
-(setq user-mail-address "ahei0802@gmail.com")
-(setq user-full-name    "ahei")
+(setq user-mail-address "weidong.lian@gmail.com")
+(setq user-full-name    "lianweidong")
 
 (setq-default default-directory "~")
 
Index: my-lisps/lisp-interaction-mode-settings.el
===================================================================
--- my-lisps/lisp-interaction-mode-settings.el	(revision 1384)
+++ my-lisps/lisp-interaction-mode-settings.el	(working copy)
@@ -2,7 +2,7 @@
 
 ;; Author: ahei <ahei0802@gmail.com>
 ;; URL: http://code.google.com/p/dea/source/browse/trunk/my-lisps/lisp-interaction-mode-settings.el
-;; Time-stamp: <2010-04-14 09:08:18 Wednesday by ahei>
+;; Time-stamp: <2011-03-09 14:55:43 Wednesday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -23,7 +23,7 @@
   "Settings for `lisp-interaction-mode'."
   (eal-define-keys
    'lisp-interaction-mode-map
-   `(("C-j" goto-line)
+   `(("C-M-j" goto-line)
      ("M-j" eval-print-last-sexp))))
 
 (eval-after-load "lisp-mode"
Index: my-lisps/isearch-face-settings.el
===================================================================
--- my-lisps/isearch-face-settings.el	(revision 1384)
+++ my-lisps/isearch-face-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-04-05 17:06:00 Monday by ahei>
+;; Time-stamp: <2011-01-28 09:18:32 Friday by lian>
 
 ;; This file is free software; you can redistribute it and/or modify
 ;; it under the terms of the GNU General Public License as published by
@@ -23,7 +23,7 @@
   (set-face-background 'isearch "blue")
   (when (not is-before-emacs-21)
     (set-face-foreground 'lazy-highlight "black")
-    (set-face-background 'lazy-highlight "white"))
+    (set-face-background 'lazy-highlight "yellow"))
   (custom-set-faces '(isearch-fail ((((class color)) (:background "red"))))))
 
 (eval-after-load "isearch"
Index: my-lisps/auto-complete-settings.el
===================================================================
--- my-lisps/auto-complete-settings.el	(revision 1384)
+++ my-lisps/auto-complete-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-12-03 16:55:07 Friday by taoshanwen>
+;; Time-stamp: <2011-03-02 18:31:21 Wednesday by lian>
 
 (require 'auto-complete-config)
 (require 'auto-complete+)
@@ -189,6 +189,15 @@
           ac-source-files-in-current-dir
           ac-source-filename)))
 
+(defun ac-settings-4-latex ()
+  (setq ac-sources
+        '(ac-source-yasnippet
+          ac-source-abbrev
+          ac-source-words-in-buffer
+          ac-source-words-in-same-mode-buffers
+          ac-source-files-in-current-dir
+          ac-source-filename)))
+
 (am-add-hooks
  `(lisp-mode-hook emacs-lisp-mode-hook lisp-interaction-mode-hook
                   svn-log-edit-mode-hook change-log-mode-hook)
@@ -205,7 +214,8 @@
    ('ruby-mode-hook   'ac-settings-4-ruby)
    ('html-mode-hook   'ac-settings-4-html)
    ('awk-mode-hook    'ac-settings-4-awk)
-   ('tcl-mode-hook    'ac-settings-4-tcl)))
+   ('tcl-mode-hook    'ac-settings-4-tcl)
+   ('latex-mode-hook    'ac-settings-4-latex)))
 
 (eal-eval-by-modes
  ac-modes
Index: my-lisps/multi-term-settings.el
===================================================================
--- my-lisps/multi-term-settings.el	(revision 1384)
+++ my-lisps/multi-term-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2011-02-15 22:38:14 Tuesday by taoshanwen>
+;; Time-stamp: <2011-02-25 15:14:52 Friday by lian>
 
 (require 'multi-term)
 
@@ -70,12 +70,13 @@
  term-bind-key-alist
  `(("C-c"   . term-send-raw)
    ("C-p"   . term-send-raw)
+   ("C-u"   . cua-scroll-down)
    ("C-n"   . term-send-raw)
    ("C-s"   . isearch-forward)
    ("C-r"   . term-send-raw)
    ("C-m"   . term-send-raw)
    ("C-k"   . term-send-kill-whole-line)
-   ("C-y"   . term-send-raw)
+   ("C-y"   . term-send-yank)
    (,(if window-system "C-/" "C-_") . term-send-undo)
    ("C-M-h" . term-send-backward-kill-semi-word)
    ("M-H"   . enter-text-mode)
Index: my-lisps/yasnippet-settings.el
===================================================================
--- my-lisps/yasnippet-settings.el	(revision 1384)
+++ my-lisps/yasnippet-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-04-26 10:22:22 Monday by ahei>
+;; Time-stamp: <2011-02-23 18:11:03 Wednesday by lian>
 
 (require 'yasnippet)
 
@@ -8,6 +8,7 @@
 
 (defun yasnippet-settings ()
   "settings for `yasnippet'."
+  (setq yas/prompt-functions '(yas/ido-prompt yas/x-prompt yas/dropdown-prompt))
   (setq yas/root-directory (concat my-emacs-path "snippets"))
 
   (defun yasnippet-unbind-trigger-key ()
@@ -33,7 +34,7 @@
 
 (eal-define-keys
  'yas/minor-mode-map
- `(("C-c C-f" yas/find-snippets)))
+ `(("C-c F" yas/find-snippets)))
 
 (eval-after-load "yasnippet"
   `(yasnippet-settings))
Index: my-lisps/dev-settings.el
===================================================================
--- my-lisps/dev-settings.el	(revision 1384)
+++ my-lisps/dev-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2010-11-29 14:42:43 Monday by taoshanwen>
+;; Time-stamp: <2011-03-03 11:00:18 Thursday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -52,7 +52,7 @@
  `(("{" skeleton-c-mode-left-brace)))
 
 ;; 动态检查语法错误
-(require 'flymake-settings)
+;; (require 'flymake-settings)
 
 (defun skeleton-c-mode-left-brace (arg)
   (interactive "P")
@@ -92,15 +92,15 @@
 ;; cedet 强大的开发工具, 包括代码浏览, 补全, 类图生成
 ;; 用CEDET浏览和编辑C++代码 http://emacser.com/cedet.htm
 ;; Emacs才是世界上最强大的IDE － cedet的安装 http://emacser.com/install-cedet.htm
-(require 'cedet-settings)
+;; (require 'cedet-settings)
 
 ;; ecb 代码浏览器
-(require 'ecb-settings)
+;; (require 'ecb-settings)
 
 ;; doxygen
-(require 'doxymacs-settings)
+;; (require 'doxymacs-settings)
 
-(require 'autoconf-mode-settings)
+;; (require 'autoconf-mode-settings)
 
 ;; 像Eclipse那样高亮光标处单词
 (require 'highlight-symbol-settings)
@@ -174,6 +174,7 @@
      (message (concat "Wrote " (buffer-file-name))))))
 
 ;; 显示变量, 函数的声明
+
 (require 'eldoc-settings)
 
 ;; 自动给你加上括号
@@ -211,7 +212,7 @@
                  zjl-hl-settings
                  ;; 把speedbar放到当前frame里面
                  sr-speedbar-settings
-                 codepilot-settings
+                 ;; codepilot-settings
                  perl-settings
                  ;; 实现程序变量的自动对齐
                  align-settings
