Index: .emacs
===================================================================
--- .emacs	(revision 1400)
+++ .emacs	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2011-04-30 21:30:38 Saturday by taoshanwen>
+;; Time-stamp: <2011-06-16 21:39:10 Thursday by lian>
 
 (defconst my-emacs-path           "~/emacs/" "我的emacs相关配置文件的路径")
 (defconst my-emacs-my-lisps-path  (concat my-emacs-path "my-lisps/") "我自己写的emacs lisp包的路径")
@@ -12,8 +12,11 @@
 (my-add-subdirs-to-load-path my-emacs-lisps-path)
 (my-add-subdirs-to-load-path my-emacs-my-lisps-path)
 
+;; lian-preload
+(require 'my-preload)
+
 ;; 一个Emacs的16进制文件查看器，可以瞬间打开巨大的文件，比官方的hexl-mode好用
-(require 'hexview-mode)
+;; (require 'hexview-mode)
 
 ;; 一些基本的小函数
 (require 'ahei-misc)
@@ -62,7 +65,7 @@
 
 ;; 打开压缩文件时自动解压缩
 ;; 必须放在session前面
-(auto-compression-mode 1)
+;; (auto-compression-mode 1)
 
 ;; 所有关于buffer方面的配置
 (require 'all-buffer-settings)
@@ -137,7 +140,7 @@
 
 ;; Emacs可以做为一个server, 然后用emacsclient连接这个server,
 ;; 无需再打开两个Emacs
-(require 'emacs-server-settings)
+;; (require 'emacs-server-settings)
 
 (require 'apropos-settings)
 (require 'completion-list-mode-settings)
@@ -297,7 +300,7 @@
 
 ;; erc: Emacs中的IRC
 ;; ERC使用简介 emacser.com/erc.htm
-(require 'erc-settings)
+;; (require 'erc-settings)
 
 ;; spell check
 (setq-default ispell-program-name "aspell")
@@ -319,9 +322,9 @@
   (add-hook 'kill-emacs-hook 'install-.emacs))
 
 ;; 把pdf,ps,dvi文件转换为png格式, 在Emacs里面浏览
-(if is-after-emacs-23
-    (require 'doc-view)
-  (setq doc-view-conversion-refresh-interval 3))
+;; (if is-after-emacs-23
+;;   (require 'doc-view)
+;;   (setq doc-view-conversion-refresh-interval 3))
 
 ;; 在Emacs里面使用shell
 (require 'term-settings)
@@ -332,7 +335,7 @@
 (require 'ioccur)
 
 ;; 查询天气预报
-(require 'weather-settings)
+;; (require 'weather-settings)
 
 ;; 可以把光标由方块变成一个小长条
 (require 'bar-cursor)
@@ -410,8 +413,8 @@
 (require 'todochiku-settings)
 
 ;; twitter client
-(require 'eagle-settings)
-(require 'twit-settings)
+;; (require 'eagle-settings)
+;; (require 'twit-settings)
 
 ;; 模拟vi的点(.)命令
 (require 'dot-mode)
@@ -437,11 +440,11 @@
 (require 'org-settings)
 
 ;; 强大的发布工具
-(require 'muse-settings)
+;; (require 'muse-settings)
 
 ;; 用weblogger写WordPress博客
 ;; http://emacser.com/weblogger.htm
-(require 'weblogger-settings)
+;; (require 'weblogger-settings)
 
 ;; 关闭buffer的时候, 如果该buffer有对应的进程存在, 不提示, 烦
 (delq 'process-kill-buffer-query-function kill-buffer-query-functions)
@@ -480,7 +483,7 @@
  `(("C-x M-O" open-current-file-with-app)))
 
 ;; 一些有趣的东西
-(require 'funny)
+;; (require 'funny)
 
 ;; 模拟vi的f命令, 非常好用
 (require 'go-to-char)
@@ -518,4 +521,4 @@
 (sb-update)
 
 ;; 加载用户自己的配置
-(load "~/my.emacs" t)
+(load "~/my.emacs" t) 
Index: my-lisps/cedet/semantic/cedet-semantic-settings.el
===================================================================
--- my-lisps/cedet/semantic/cedet-semantic-settings.el	(revision 1400)
+++ my-lisps/cedet/semantic/cedet-semantic-settings.el	(working copy)
@@ -2,7 +2,7 @@
 
 ;; Author: ahei <ahei0802@gmail.com>
 ;; URL: http://code.google.com/p/dea/source/browse/trunk/my-lisps/cedet-semantic-settings.el
-;; Time-stamp: <2010-11-30 11:42:06 Tuesday by taoshanwen>
+;; Time-stamp: <2011-02-18 18:54:57 Friday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -21,7 +21,8 @@
 
 (semantic-load-enable-excessive-code-helpers)
 (global-semantic-idle-completions-mode -1)
-(semantic-load-enable-semantic-debugging-helpers)
+;; (semantic-load-enable-semantic-debugging-helpers)
+;; (semantic-load-enable-all-exuberent-ctags-support) 
 (when window-system
   (global-semantic-tag-folding-mode 1))
 
@@ -35,6 +36,7 @@
    `(c-mode-base-map makefile-gmake-mode-map python-mode-map perl-mode-map sh-mode-map)
    `(("C-c C-j" semantic-ia-fast-jump)
      ("C-c j"   semantic-complete-jump-local)
+     ("C-c u" semantic-debug-idle-work-function)
      ("C-c n"   senator-next-tag)
      ("C-c p"   senator-previous-tag)
      ("C-c M-m" semantic-goto-local-main)
@@ -98,7 +100,18 @@
    'emaci-mode-map
    `(("." emaci-.)
      ("," emaci-\,)))
+
+  ;; semantic persistent-path  
+  (setq semanticdb-persistent-path (append semanticdb-persistent-path lian-source-file-dir))
   
+  ;; my include path
+  (let ((include-dirs lian-head-file-dir))
+    (mapc (lambda (dir)
+            (semantic-add-system-include dir 'c++-mode)
+            (semantic-add-system-include dir 'c-mode))
+          include-dirs))
+  
+  
   ;; system include path
   (if (or mswin cygwin)
       (dolist (mode '(c-mode c++-mode))
Index: my-lisps/dev-settings.el
===================================================================
--- my-lisps/dev-settings.el	(revision 1400)
+++ my-lisps/dev-settings.el	(working copy)
@@ -1,6 +1,6 @@
 ;; -*- Emacs-Lisp -*-
 
-;; Time-stamp: <2011-03-20 16:58:35 Sunday by taoshanwen>
+;; Time-stamp: <2011-03-03 11:00:18 Thursday by lian>
 
 ;; This  file is free  software; you  can redistribute  it and/or
 ;; modify it under the terms of the GNU General Public License as
@@ -52,7 +52,7 @@
  `(("{" skeleton-c-mode-left-brace)))
 
 ;; 动态检查语法错误
-(require 'flymake-settings)
+;; (require 'flymake-settings)
 
 (defun skeleton-c-mode-left-brace (arg)
   (interactive "P")
@@ -92,15 +92,15 @@
 ;; cedet 强大的开发工具, 包括代码浏览, 补全, 类图生成
 ;; 用CEDET浏览和编辑C++代码 http://emacser.com/cedet.htm
 ;; Emacs才是世界上最强大的IDE － cedet的安装 http://emacser.com/install-cedet.htm
-(require 'cedet-settings)
+;; (require 'cedet-settings)
 
 ;; ecb 代码浏览器
-(require 'ecb-settings)
+;; (require 'ecb-settings)
 
 ;; doxygen
-(require 'doxymacs-settings)
+;; (require 'doxymacs-settings)
 
-(require 'autoconf-mode-settings)
+;; (require 'autoconf-mode-settings)
 
 ;; 像Eclipse那样高亮光标处单词
 (require 'highlight-symbol-settings)
@@ -174,6 +174,7 @@
      (message (concat "Wrote " (buffer-file-name))))))
 
 ;; 显示变量, 函数的声明
+
 (require 'eldoc-settings)
 
 ;; 自动给你加上括号
@@ -211,13 +212,13 @@
                  zjl-hl-settings
                  ;; 把speedbar放到当前frame里面
                  sr-speedbar-settings
-                 codepilot-settings
+                 ;; codepilot-settings
                  perl-settings
                  ;; 实现程序变量的自动对齐
                  align-settings
                  ;; 生成c程序调用图
                  ;; http://emacser.com/emacs-cflow.htm
-                 cflow-mode-settings
-                 python-settings))
+                 cflow-mode-settings))
+                 ;;python-settings))
 
 (provide 'dev-settings)
